app/views/accounts/_objective.html.slim

```diff
@@ -4,4 +4,5 @@ tr
   td= number_to_percentage(account.balance / objective.amount * 100, precision: 0)
   td= ([objective.amount - account.balance, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).round(2)
   td= image_tag objective.image_url
-  td= link_to :delete, objective_path(objective), method: :delete, data: {confirm: 'Delete this objective?'}
+  - unless read_only?(account.id)
+    td= link_to :delete, objective_path(objective), method: :delete, data: {confirm: 'Delete this objective?'}
```

Comment:
we should generate the constant number of td tags. so, if read-only the tag should be still there, but tiit should be empty